bom/pom.xml

```diff
@@ -264,6 +284,26 @@
                 <artifactId>cxf-rt-rs-service-description</artifactId>
                 <version>${project.version}</version>
             </dependency>
+            <dependency>
+                <groupId>org.apache.cxf</groupId>
+                <artifactId>cxf-rt-rs-service-description-openapi-v3</artifactId>
+                <version>${project.version}</version>
+            </dependency>
+                        <dependency>
```

Comment:
The `cxf-rt-rs-service-description-common-openapi` should probably not be there as well, it is more or less internal to CXF implementation

Reply:
Removed cxf-rt-rs-service-description-common-openapi